SPECS/pam_krb5/pam_krb5.spec

```diff
@@ -0,0 +1,72 @@
+Summary:        A Pluggable Authentication Module for Kerberos 5
+Name:           pam_krb5
+Version:        4.11
+Release:        2%{?dist}
+License:        MIT AND BSD
```

Comment:
This seems to contradict the [README](https://github.com/rra/pam-krb5#license), which mentions dual licensing under BSD/GPL+

Reply:
Good catch! I checked the history and changed it back to the previous license.